README.md

```diff
@@ -1,9 +1,76 @@
-# jaeger-mongodb
+# jaeger-mongodb 
 
+[jaeger]: https://github.com/jaegertracing/jaeger/
```

Comment:
can we move all the link anchors to the bottom of the readme.

Reply:
fixed